spec/adapter/integration_spec.rb

```diff
@@ -91,7 +91,7 @@
       if Sidekiq::VERSION < '4'
         Sidekiq::Scheduled::Poller.new.poll
       else
-        Sidekiq::Scheduled::Poller.new.enqueue
+        Sidekiq::Scheduled::Poller.new(Sidekiq).enqueue
```

Comment:
I'm confused on this one, in the source code it looks like this is supposed to accept a `Sidekiq::Config` instance.

Reply:
Also: if we're running CI against all the sidekiq versions this gem supports (although maybe we should change that, it supports some really old versions), the change to `Poller#initialize` to accept an argument wasn't until v6.5.0

Reply:
Ah lol, this doesn't have any CI right now, does it? Maybe we should add github actions for that with matrix of the ruby/sidekiq support before changing more things?